src/navbar/types.ts

```diff
@@ -134,4 +136,6 @@ export interface NavbarProps<T = void> extends NavbarSharedProps {
     /** Specifies if masthead should be rendered */
     masthead?: boolean | undefined;
     layout?: "default" | "stretch" | undefined;
+    /** Custom aria-label for the main navigation header */
+    headerLabel?: string | undefined;
```

Comment:
include `drawerLabel` here as well as this is the entry point for all props

make sure to forward it to `<Drawer>` in `navbar.tsx`